### DISCUSSION TITLE
upgrading ~0.16.5 to 0.18.1 Release

### DISCUSSION THREAD

**Discussion by HEAVYPOLY** [pocketbase/pocketbase#3266] [Q&A]
About to upgrade my production app from ~0.16.5 to 0.18.1, is there anything needed to prepare for this or should be ok to go ahead?

**Comment by ganigeorgiev** [comment]
There are no major breaking changes to the web APIs in v0.17 and v0.18 but there are some breaking changes to the internals, so if you use PocketBase as framework, make sure to check the bullet points from the changelog marked with "⚠️":

- https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#v0170
- https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#v0180

**Reply by HEAVYPOLY** [reply]
Great to hear, will upgrade then.  Not using as framework yet but plan to once upgraded, thank you.
